src/opls.js

```diff
@@ -1,6 +1,5 @@
-'use strict';
 
-var Matrix = require('ml-matrix').Matrix;
+/*var Matrix = require('ml-matrix').Matrix;
```

Comment:
if the code is not usable is better to move it to a different branch and deleted from here